src/plugins/management/public/components/management_sidebar_nav/management_sidebar_nav.tsx

```diff
@@ -17,184 +17,90 @@
  * under the License.
  */
 
-import {
-  EuiIcon,
-  // @ts-ignore
-  EuiSideNav,
-  EuiScreenReaderOnly,
-} from '@elastic/eui';
-import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiIcon, EuiSideNav, EuiScreenReaderOnly, EuiSideNavItemType } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import React, { ReactElement } from 'react';
-import { LegacySection, LegacyApp } from '../../types';
-import { ManagementApp } from '../../management_app';
-import { ManagementSection } from '../../management_section';
+import React, { useState } from 'react';
+import { AppMountParameters } from 'kibana/public';
+import { ManagementApp, ManagementSection } from '../../utils';
 
-interface NavApp {
-  id: string;
-  name: ReactElement | string;
-  [key: string]: unknown;
-  order: number; // only needed while merging platform and legacy
-}
+import './_sidebar_nav.scss';
```

Comment:
Same here, but also the name of the SASS file needs to match that of the JS file.

Reply:
done